documentation/AuthorizationService.md

```diff
@@ -1,123 +1,245 @@
+#### grantGpiiAppInstallationAuthorization(gpiiToken, clientId)
+* **description**: Grant an authorization to a GPII app installation. The authorization allows a GPII app installation to access user settings associated with the given GPII token. The scenarios are handled by this function: 
+    * If the given GPII app installation has never been assigned an authorization for the given GPII token, the function will generate and return one.
+    * If the given GPII app installation has already been assigned an authorization for the given GPII token, and this token has not expired, the function will return this existing authorization;
+    * If the given GPII app installation has already been assigned an authorization for the given GPII token, but this token has already expired, the fuction will generate and return a new one.
 * **parameters:** 
-    * userId: Number. A system generated unique number that identifies the user.
-    * clientId: Number. A system generated unique number that identifies the client.
-    * redirectUri: String. The client redirection URI that the authorization server directs the user-agent to when a authorization decision is established.
-    * selectedPreferences: Object. An object specifying the preferences that the user has selected to share, in the privacy ontology. An example:
+    * gpiiToken: String. A GPII token that associates with user preferences.
+    * clientId: String. A system generated unique string that identifies the client.
+* **return:** Object. Contains an access token and the number of seconds that the access token will expire. For example:
+```
+{
+    "accessToken": "8ea3457bf283db5d34ea5a4079fa36b2",
+    "expiresIn": 3600
+}
+```
+Return an object that contains the error message and http status code if,
+    * The GPII token is wrong, or,
+    * The client is not a GPII App Installation Client.
+
+#### grantPrivilegedPrefsCreatorAuthorization(clientId, scope)
+* **description**: Grant an authorization to a privileged preferences creator. This authorization allows a privileged preferences creator to add new preferences sets. The scenarios handled by this function: 
+    * If the given privileged preferences creator has never been assigned an authorization, the function will generate and return one.
+    * If the given privileged preferences creator has already been assigned an authorization,the function will return this existing authorization.
+* **parameters:** 
+    * clientId: String. A system generated unique number that identifies the client.
+        - Notes: "oauth2ClientCredentials" is the only OAuth2 client type that is allowed to request privileged prefs creator authorizations. See [the `Client` document structure](https://github.com/GPII/universal/blob/master/documentation/AuthServer.md#client).
```

Comment:
I believe that this note and the link to the `Client` document structure are now out of date?